pages/contribution-flow.js

```diff
@@ -134,6 +138,9 @@ class NewContributionFlowPage extends React.Component {
     }
 
     const contributionBLocker = getContributionBlocker(LoggedInUser, account, tier, Boolean(this.props.tierId));
+    if (contributionBLocker.reason === CONTRIBUTION_BLOCKER.NO_CUSTOM_CONTRIBUTION) {
```

Comment:
@saideepesh000 : I don't see this `NO_CUSTOM_CONTRIBUTION` being defined anywhere. Hence I think this should fail. This is probably why the E2E tests are failing. 🤔

Reply:
I actually imported `CONTRIBUTION_BLOCKER` from [ContributionBlocker.js](https://github.com/opencollective/opencollective-frontend/blob/3f25bbbcb324d58747927a6f5a450b6206bc5f1b/components/contribution-flow/ContributionBlocker.js#L12)

Reply:
@saideepesh000 : Ops my mistake. But still the E2E tests are failing since the `/donate` url seems to error out in the server. 

https://opencollective-frontend-b1gll8btb-opencollective.vercel.app/apex/donate